### PR TITLE
igv 2.4.16

### PR DESCRIPTION
**Pull request by sjackman**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

**Comment by sjackman** [comment]
`brew test igv` failed. I'm unable to reproduce this failure locally.
```
==> /usr/local/Cellar/igv/2.4.16/bin/igv -b script
Picked up _JAVA_OPTIONS:  -Duser.home=/Users/brew/Library/Caches/Homebrew/java_cache
Error: igv: failed
An exception occurred within a child process:
  Test::Unit::AssertionFailedError: </Version/> expected to be =~
<"WARN [2018-12-04T18:41:13,754]  [Globals.java:138] [AWT-EventQueue-0]  Development mode is enabled\nERROR [2018-12-04T18:41:13,793]  [DirectoryManager.java:137] [AWT-EventQueue-0]  Failed to create user directory!\nERROR [2018-12-04T18:41:16,525]  [DefaultExceptionHandler.java:49] [AWT-EventQueue-0]  Unhandled exception\njava.awt.HeadlessException: null\n\tat java.awt.GraphicsEnvironment.checkHeadless(GraphicsEnvironment.java:204)
```
https://jenkins.brew.sh/job/Homebrew%20Core%20Pull%20Requests/34553/version=mojave/testReport/junit/brew-test-bot/mojave/test_igv/

**Comment by apjanke** [comment]
This might be the relevant exception in the stack trace:

```
Unhandled exception
java.awt.HeadlessException
```

That ["HeadlessException"](https://docs.oracle.com/javase/7/docs/api/java/awt/HeadlessException.html) means it's trying to do some AWT graphics stuff that expects a graphical desktop session. Makes sense since the CI environment is running on a VM on a server.

Unfortunately, I don't see any `--version` options or similar for an alternate test.

**Comment by sjackman** [comment]
This test passed in previous versions, but it's quite possible that either Java or IGV has changed so that this test no longer works.

What have we done in the past with tools that require a graphical session?

**Comment by sjackman** [comment]
I've reported this issue upstream: https://github.com/igvteam/igv/issues/611
Here are the command line options of IGV: https://software.broadinstitute.org/software/igv/startingIGV

**Comment by sjackman** [comment]
Rebased onto master to give it another go.

File: Formula/igv.rb [hunk 1]
```diff
@@ -15,7 +15,11 @@ def install
...
 
   test do
-    (testpath/"script").write "exit"
-    assert_match "Version", shell_output("#{bin}/igv -b script")
+    assert_match "org/broad/igv/ui/IGV.class", shell_output("jar tf #{libexec}/lib/igv.jar")
+    # Fails on Jenkins with Unhandled exception: java.awt.HeadlessException
+    unless ENV["CI"]
+      (testpath/"script").write "exit"
+      assert_match "Version", shell_output("#{bin}/igv -b script")
+    end
```
Comment (sjackman) [inline Formula/igv.rb]: @fxcoudert Any thoughts on this hack? The previous test is failing on Jenkins with `Unhandled exception: java.awt.HeadlessException`. The test succeeds on my local machine. I've disabled this test on CI, and added a fallback test.

**Pull request closed without merging**